### DISCUSSION TITLE
i'm ousted

### DISCUSSION THREAD

**Discussion by sa2ajj** [buildbot/buildbot#7452] [General]
any discussion to link to?

**Comment by sa2ajj** [comment]
apparently there's a difference b/w current contributors and the ones from 20 yrs ago

**Reply by bdbaddog** [reply]
some more detail would be helpful here..

**Reply by p12tic** [reply]
@sa2ajj Could you provide more detail? What you see here on Github is a complete picture of the project. There's no secret discussions channel for blessed contributors. You have exactly the same permissions as e.g. me.

**Reply by bdbaddog** [reply]
What's going on here?

**Reply by sa2ajj** [reply]
nothing special, i think. probably my loss of my previous email address resulted in the changes

we locally use the latest Buildbot, though there were no changes that are worth contributing

**Reply by sa2ajj** [reply]
sorry for a tone that could have been misinterpreted as frustration though it was just a surprise of sort. please let's conclude here. no bad feelings here though as i re-read my original message could be interpreted otherwise

**Reply by verm** [reply]
Okay no problem we're just trying to figure out what's going on because nothing has changed. :)

**Comment by sa2ajj** [comment]
i think i'll close the discussion. my previous contributions are no more 😊

**Reply by verm** [reply]
What do you mean?  It says Maintainer right beside your name?  What contributions are you referring to?

<img width="361" height="41" alt="image" src="https://github.com/user-attachments/assets/833d2c51-b130-4ab8-8b82-ac2e122d5d5f" />
